app/pb_kits/playbook/pb_textarea/_textarea.scss

```diff
@@ -2,9 +2,9 @@
 @import "./textarea_mixin";
 
 [class^=pb_textarea_kit] {
+  margin: 8px;
   margin-bottom: $space_sm;
```

Comment:
Looks like these are still incorrect. Did we revert?

Reply:
I accidentally overwrote some of Terry's changes.  Good catch.  I just pushed up a fix